README.md

```diff
@@ -1,2 +1,40 @@
 # target_seq_bed_files
 A repository to capture genes targeted by WES and target-Seq from ICGC-ARGO projects
+
+## Purpose of field and definitions
+
+To support WXS and Targeted Sequencing the following fields are now required :
+
+- "Target_capture_kit"
+- "primary_target_regions"
+- "capture_target_regions"
+
+### Target_capture_kit: 
+The experimental kit (commercial/lab owned) used to isolate specific regions of the genome. RNA bait sequences bind to relevant complementary DNA sequences. Used in combination with WGS, this allows for focused sequencing to improve depth/coverage at regions of interest.
+###  Primary_target_regions:
+A bed file of biologically relevant regions targeted by Target_capture_kit. These regions will typically extend capture_target_regions. Synonyms include : “empirical target”,”regions” and or “target intervals”
+###  Capture_target_regions:
+A bed file of regions captured by probes. “Covered regions” and “bait intervals”
+## Specifications of Bed Files
+Bed files are text based with three columns: chromosome, chromosome start and chromosome end
+Coordinates are to be zero-based and based on Grch38/Hg38
+Bed files will hosted on www.github.com/icgc-argo-workflows/target_seq_bed_files
+
```

Comment:
## What are Targeted Sequencing BED Files?
Targeted Sequencing BED Files are plain text files that specify genomic regions of interest for targeted sequencing experiments. They are formatted according to the BED (Browser Extensible Data) file format, a simple and flexible standard widely used in genomics.

### BED File Format
A BED file lists genomic intervals using one line per region, with fields separated by tabs. The standard BED format includes three required fields:
- chrom: Chromosome name (e.g., chr1, chr2, ..., chrX, chrY, chrM)
- chromStart: Start position of the genomic interval (0-based, inclusive, GRCh38/Hg38)
- chromEnd: End position of the genomic interval (0-based, exclusive, GRCh38/Hg38)

*Note*: The chromStart position is zero-based and inclusive, while the chromEnd position is zero-based and exclusive. This means the first base of a chromosome is position 0, and the interval includes bases from chromStart up to but not including chromEnd.

### Example of a BED File Entry
```
chr7    55019017    55019067 
```